packages/perseus/src/widget-ai-utils/mock-widget/prompt-utils.ts

```diff
@@ -1,5 +1,5 @@
-import type {PerseusMockWidgetUserInput} from "../../widgets/mock-widgets/mock-widget";
 import type mockWidget from "../../widgets/mock-widgets/mock-widget";
+import type {PerseusMockWidgetUserInput} from "../../widgets/mock-widgets/mock-widget-types";
```

Comment:
Why do we need AI prompt utils for a mock widget?

Reply:
I think just to "round out" the implementation of the mock widget. I don't think we actually use it.  Perhaps it'd be better if we deleted this to avoid confusion and have folks think it serves some purpose.

Reply:
Changed my mind. There's tests that exercise the `Renderer` calling the widget's `getPromptJSON()` using the mock widget. I'll leave it in place as that feels like a valid use of the mock widget. I'd also like to avoid doing this sort of refactoring in this PR as I'm trying to stick to just dealing with after effects from merging `main`. :)